packages/twenty-front/src/modules/object-record/record-table/record-table-section/components/RecordTableRecordGroupSection.tsx

```diff
@@ -32,6 +32,11 @@ const StyledChevronContainer = styled(RecordTableTd)`
   vertical-align: middle;
 `;
 
+const StyledAnimatedLightIconButton = styled(AnimatedLightIconButton)`
+  display: block;
```

Comment:
this looks anti pattern to me to change the display of a component. Why do we need that?

Reply:
We need it in order to properly align the button component as this one is placed in a table `td` component.
So we can just apply `vertical-align: middle` and `text-align: center` but it's not enough.
Applying this style on the button allow the container css to work properly.

Reply:
ok, I think this should be exceptional and we should avoid changing a component behavior from the outside (this will result in unconsistent behaviors across the product)